packages/server/src/app/Application.js

```diff
@@ -522,7 +518,9 @@ export class Application extends Koa {
         }
         this.use(emitUserEvents())
       }
-      // 6. finally handle the found route, or set status / allow accordingly.
+      // 6. Attach a logger instance with the user to the context.
+      this.use(attachUserToLog())
```

Comment:
Shouldn't this be inside the `if (app.passport) {}` block? Without passport, no user?